src/Sylius/Bundle/UiBundle/Resources/assets/sass/_variables.scss

```diff
@@ -0,0 +1 @@
+$sylius: #1abb9c;
```

Comment:
I suggest a naming convention for variables that helps identify them explicitly, like `$colour-sylius-green`. 

Then do what you've done and name variables relative to what they represent like:

```
$link-color: $sylius-colour-green
$main-background-color: lighten($sylius-colour-green, 5%);
$off-background-color: lighten($sylius-colour-green, 10%;
```

The variable names you use in CSS then represent the areas themselves and you keep your colour palette separate.